cluster/proto/api/rbac_requests.go

```diff
@@ -15,8 +15,21 @@ import (
 	"github.com/weaviate/weaviate/usecases/auth/authorization"
 )
 
+const (
+	// in case changes happens to the RBAC message, add new version before the RBACLatestCommandPolicyVersion
+	// RBACCommandPolicyVersionV0 represents the first version of RBAC commands
+	RBACCommandPolicyVersionV0 = iota
+
+	// RBACLatestCommandPolicyVersion represents the latest version of RBAC commands policies
+	// It's used to migrate policy changes. if we end up with a cluster having different version
+	// that won't be a problem because the version here is not about the message change but more about
+	// the content of the body which will dumbed anyway in RBAC storage.
+	RBACLatestCommandPolicyVersion
+)
```

Comment:
Just to be sure I understand correctly:

replayes of old `cmd.CreateRolesRequest` will not contain the version parameter and go will use the default value of 0 (==RBACCommandPolicyVersionV0). But for new commands we set it to 1 (==RBACLatestCommandPolicyVersion) to be able to distinguish between new and old?

Reply:
exactly,  the message body won't have version 
-  it will be default 0 
-  new ones will have version 1